tests/server.go

```diff
@@ -0,0 +1,49 @@
+package tests
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func NewHTTPServerPod(port int) *corev1.Pod {
+	serverCommand := fmt.Sprintf(`echo -e "HTTP/1.1 200 OK\nContent-Length: 11\n\nHello World!" | nc -kvlp %d`, port)
```

Comment:
This is wrong. It will only work for the **first** probe. Sub-subsequent probes will just hang then timeout - the `echo` is only performed once. 

You didn't notice it because readiness probes only need to reply **once**.

Reply:
Please update the server command to use:

```
serverCommand := fmt.Sprintf("nc -klp %d --sh-exec 'echo -e \"HTTP/1.1 200 OK\\nContent-Length: 11\\n\\nHello World!\"'", port)`
```
which does work consistently, and provides correct line escaping.